scripts/test-tls13-dhe-shared-secret-padding.py

```diff
@@ -221,19 +226,35 @@ def main():
                     print("Got shared secret with {0} most significant "
                           "bytes equal to zero."
                           .format(min_zeros))
-                    break_loop = True
+                    break_shared = True
+                # ECDSA key shares have a constant first byte indicating
+                # the point encoding
+                if "secp" in c_name:
```

Comment:
That makes it very hard to figure when it will get here. Common logic helps, but without it, it is not possible to check when it will get here by only looking this file.

Reply:
I'm not sure if I follow... what change you suggest?

Reply:
Not a big deal, searching for an algorithm with a "regex' looked like a hack, and as in this file the strings set are similarly to: `"TLS 1.3 with {0}".format(GroupName.toStr(group))`, I could not verify that the secp will be in the actual expected name. Why not pass the type of the test?

Reply:
because at this point the test has already been executed - I only have access to the name, decision tree and result; if possible, I'd rather not modify root node to carry additional data – it will make future reuse of tests for fuzzing more complex

I prefer to handle them the same to the point it is possible as the ffdhe and ecdhe key exchange is not handled differently from protocol level